examples/README.md

```diff
@@ -0,0 +1,11 @@
+The scripts in this directory provide code examples using Confluent's Python client:
+
+* [adminapi.py](adminapi.py): library of Kafka AdminClient operations
```

Comment:
"library" implies that this is a library/module, which it is not. Maybe "collection" instead?

Reply:
"Collection of Kafka Admin API operations" is probably better, since it is more descriptive of what it is.

Reply:
Fixed